sample/src/androidTest/java/com/schibsted/spain/barista/sample/SeekBarAssertionsTest.java

```diff
@@ -0,0 +1,34 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.assertion.BaristaSeekBarAssertions.assertProgress;
+import static com.schibsted.spain.barista.assertion.BaristaSeekBarAssertions.assertProgressIsMax;
+import static com.schibsted.spain.barista.assertion.BaristaSeekBarAssertions.assertProgressIsMin;
+
+@RunWith(AndroidJUnit4.class)
+public class SeekBarAssertionsTest {
+
+  @Rule
+  public ActivityTestRule<SomeSeekBarsWithDifferentProgressesActivity> activityRule =
+      new ActivityTestRule<>(SomeSeekBarsWithDifferentProgressesActivity.class);
+
+  @Test
+  public void checkProgress() {
+    assertProgress(R.id.seek_bar_progress_27, 27);
+  }
+
+  @Test
+  public void checkMinProgress() {
+    assertProgressIsMin(R.id.seek_bar_min_progress);
+  }
+
+  @Test
+  public void checkMaxProgress() {
```

Comment:
Can you also add the checks that the assertion fails if the data is not the expected one? Something like I did on this commit of another PR: https://github.com/SchibstedSpain/Barista/pull/170/commits/fc159d8798df422ab88a7834410f8db6193f37fb

Reply:
It makes sense 👍